core/runtime/jsruntime.rs

```diff
@@ -3,7 +3,10 @@ use super::bindings;
 use super::bindings::watch_promise;
 use super::exception_state::ExceptionState;
 use super::jsrealm::JsRealmInner;
+use super::op_driver::OpDriver;
 use super::snapshot_util;
+use crate::OpMetricsEvent;
+use crate::_ops::dispatch_metrics_async;
```

Comment:
Is it on purpose that this is now imported from `crate::_ops` and not `crate::ops_metrics`?

Reply:
Nope, `rust-analyzer` picked the wrong import site, will fix.